kafka/setup-key-mhsm.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```

Comment:
https://github.com/microsoft/confidential-sidecar-containers/blob/main/examples/aasp/setup-key-mhsm.sh

It would be better to link the user out to this file since it will be updated if things change.

Reply:
this file was renamed. Doc was updated